transcom/mymove
# B-17856-INT remove sit-delivery endpoint from Prime API

## [B-17856 ](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A865118)

## Summary
Removes the  /mto-shipments/{mtoShipmentID}/sit-delivery Prime API end point that was added as part of B-18028 as this endpoint turned out not to be needed. The logic for calculating the Authorized End Date and Required Delivery Date will be added to another end point in B-17854.

Is there anything you would like reviewers to give additional scrutiny?

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Open Prime API swagger page. http://primelocal:3000/swagger-ui/prime.html
2. Verify that the /mto-shipments/{mtoShipmentID}/sit-delivery endpoint is no longer there
3. Run make server_tests and make sure all tests pass

### Backend


## Screenshots
